src/watertap_contrib/seto/unit_models/surrogate/lt_med_surrogate.py

```diff
@@ -687,86 +683,6 @@ def calculate_scaling_factors(self):
-
-        sf = iscale.get_scaling_factor(
-            self.cooling_out_props[0].conc_mass_phase_comp["Liq", "TDS"]
-        )
-        iscale.constraint_scaling_transform(self.eq_cooling_salinity, sf)
-
-        sf = iscale.get_scaling_factor(self.brine_props[0].temperature)
-        iscale.constraint_scaling_transform(self.eq_brine_temp, sf)
-
-        sf = iscale.get_scaling_factor(
-            self.brine_props[0].conc_mass_phase_comp["Liq", "TDS"]
-        )
-        iscale.constraint_scaling_transform(self.eq_brine_salinity, sf)
-
-        sf = iscale.get_scaling_factor(self.feed_props[0].flow_vol_phase["Liq"])
-        iscale.constraint_scaling_transform(self.eq_dist_vol_flow, sf)
-
-        sf = iscale.get_scaling_factor(self.brine_props[0].flow_vol_phase["Liq"])
-        iscale.constraint_scaling_transform(self.eq_brine_vol_flow, sf)
-
-        sf = iscale.get_scaling_factor(self.gain_output_ratio)
-        iscale.constraint_scaling_transform(self.eq_gain_output_ratio, sf)
-
-        sf = iscale.get_scaling_factor(self.specific_area_per_m3_day)
-        iscale.constraint_scaling_transform(self.eq_specific_area_per_m3_day, sf)
-
-        sf = iscale.get_scaling_factor(self.specific_area_per_kg_s)
-        iscale.constraint_scaling_transform(self.eq_specific_area_kg_s, sf)
-
-        sf = iscale.get_scaling_factor(
-            self.steam_props[0].flow_mass_phase_comp["Vap", "H2O"]
-        )
-        iscale.constraint_scaling_transform(self.eq_steam_mass_flow, sf)
-
-        sf = iscale.get_scaling_factor(self.specific_energy_consumption_thermal)
-        iscale.constraint_scaling_transform(
-            self.eq_specific_thermal_energy_consumption, sf
-        )
-
-        sf = iscale.get_scaling_factor(self.thermal_power_requirement)
-        iscale.constraint_scaling_transform(self.eq_thermal_power_requirement, sf)
-
-        sf = (
-            iscale.get_scaling_factor(self.feed_cool_mass_flow)
-            * iscale.get_scaling_factor(
-                self.cooling_out_props[0].enth_mass_phase["Liq"]
-            )
-            * 1e-3
-        )
-        iscale.constraint_scaling_transform(self.eq_feed_cool_mass_flow, sf)
-
-        sf = iscale.get_scaling_factor(self.feed_cool_vol_flow)
-        iscale.constraint_scaling_transform(self.eq_feed_cool_vol_flow, sf)
-
-        sf = (
-            iscale.get_scaling_factor(self.cooling_out_props[0].flow_vol_phase["Liq"])
-            / 3600
-        )
-        iscale.constraint_scaling_transform(self.eq_cool_vol_flow, sf)
-
-        for t in self.flowsheet().config.time:
-            sf = iscale.get_scaling_factor(self.feed_props[t].pressure)
-            iscale.constraint_scaling_transform(
-                self.eq_feed_to_distillate_isobaric[t], sf
-            )
-            iscale.constraint_scaling_transform(self.eq_feed_to_brine_isobaric[t], sf)
-            iscale.constraint_scaling_transform(self.eq_feed_to_cooling_isobaric[t], sf)
-
```

Comment:
Just want to confirm that, is the scaling of constraints not needed because all variables are already scaled?

Reply:
Pretty much-, I think the general guidance was to scale variables and leave the constraints as they will be handled by ipopt, unless there are some problematic constraints identified, in which case you could add some targeted contraint scaling transforms. In this case, I found that none were needed, at least for the cases tested.

Reply:
Furthermore, as I was probing @kurbansitterley 's issue (#51), my attempts seemed to improve when I took away constraint scaling from LT-MED.